src/radio/radio.js

```diff
@@ -82,7 +82,10 @@
       var button = radios[i].querySelector('.' + this.CssClasses_.RADIO_BTN);
       // Different name == different group, so no point updating those.
       if (button.getAttribute('name') === this.btnElement_.getAttribute('name')) {
-        radios[i]['MaterialRadio'].updateClasses_();
+        var r = radios[i]['MaterialRadio'];
+        if (typeof r !== 'undefined') {
```

Comment:
You can probably join the two lines above, since we're not using the variable for anything else. E.g. `if (typeof radios[i]['MaterialRadio'] !== 'undefined') {`.